Gemfile

```diff
@@ -72,6 +72,7 @@ group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   # Rspec testing framework
   gem 'rspec-rails'
+  gem 'rails-controller-testing'
```

Comment:
What is this for?

Reply:
It's used for this: `expect(response).to render_template('announcements/index')`

Do I leave it as it is or...?

Reply:
Just remove that then. No need to expect the template, that should be `view` I feel